packages/now-next/test/integration/index.test.js

```diff
@@ -37,9 +37,21 @@ it(
 it(
   'Should build the gip-gsp-404 example',
   async () => {
-    const {
-      buildResult: { output },
-    } = await runBuildLambda(path.join(__dirname, 'gip-gsp-404'));
+    const { buildResult } = await runBuildLambda(
+      path.join(__dirname, 'gip-gsp-404')
+    );
+    const { output, routes } = buildResult;
+
+    let handleErrorIdx = -1;
+
+    (routes || []).some((route, idx) => {
+      if (route.handle === 'error') {
+        handleErrorIdx = idx;
+        return true;
+      }
+    });
```

Comment:
```suggestion
    const handleErrorIdx = (routes || []).findIndex(r => r.handle === 'error')
```
Nit pick: you can use findIndex() instead of some() 🙂